pyfolio/plotting.py

```diff
@@ -573,6 +574,24 @@ def plot_rolling_returns(
         The axes that were plotted on.
 
 """
+    def draw_cone(returns, num_stdev, live_start_date, ax):
+        cone_df = timeseries.cone_rolling(
+            returns,
+            num_stdev=num_stdev,
+            cone_fit_end_date=live_start_date)
+
+        cone_df_fit = cone_df[cone_df.index < live_start_date]
+
+        cone_df_live = cone_df[cone_df.index > live_start_date]
+        temp_keep_days = cone_df_live.index < returns.index[-1]
```

Comment:
I agree, that I hate the temp variable, but I couldn't figure out how to be PEP8 compliant without breaking it up. I kept getting the SublimeText PEP8 flaker complaining at me.  Maybe i need to lookup how to breakup lines more effectlvely with PEP8